core-java/src/test/java/org/spine3/server/EngineShould.java

```diff
@@ -233,4 +228,77 @@ public void on(TaskAdded event, EventContext context) {
         public void on(ProjectStarted event, EventContext context) {
         }
     }
```

Comment:
Why not move these inner static classes to the bottom? They are not very interesting, I think.
Other than that, LGTM.